libafl/src/observers/stacktrace.rs

```diff
@@ -0,0 +1,319 @@
+//! the ``StacktraceObserver`` looks up the stacktrace on the execution thread and computes a hash for it for dedupe
+
+use crate::{
+    bolts::{
+        shmem::{ShMem, ShMemProvider, StdShMem, StdShMemProvider},
+        tuples::Named,
+    },
+    observers::Observer,
+    Error,
+};
+use ahash::AHasher;
+use regex::Regex;
+use serde::{Deserialize, Serialize};
+use std::{fmt::Debug, hash::Hasher, io::Read, process::ChildStderr};
```

Comment:
In the core libafl, we tend to avoid std as much code as possible should run in no_std. If you need std, you have to hide the std-dependant code behind the std feature, like here https://github.com/AFLplusplus/LibAFL/blob/main/libafl/src/executors/mod.rs#L15